src/components/NodeNameTitle/NodeNameTitle.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import classNames from "classnames";
+
+export interface INodeNameTitle {
```

Comment:
In this project I took the components folder to be only **shared** components  (used in other containers or that can be used in the future but then use generic naming). You can use DashboardContainer folder to put this component as it is used only there.